src/server/common/helpers/auth/get-bell-options.js

```diff
@@ -0,0 +1,39 @@
+import Jwt from '@hapi/jwt'
+import { config } from '#config/config.js'
+
+export function getBellOptions(oidcConfig) {
+  return {
+    provider: {
+      name: 'entra-id',
+      protocol: 'oauth2',
+      useParamsAuth: true,
+      auth: oidcConfig.authorization_endpoint,
+      token: oidcConfig.token_endpoint,
+      scope: ['openid', 'profile', 'email'],
```

Comment:
I found out there's an additional scope needed to make Entra ID return the `refresh token`

```js
      scope: [
        'openid',
        'profile',
        'email',
        'offline_access', // needed to ensure Entra ID returns a refreshToken
      ],
```

which I think we should add in this PR (given the `refreshToken` is included in the session data created in the `/auth/callback` handler

Reply:
I intended to leave token refresh out of this PR but forgot to remove some of the related code. I've removed it now. 
Is it ok if we pick all this up in the next PR?